packages/features/bookings/Booker/components/AvailableTimeSlots.tsx

```diff
@@ -114,6 +126,8 @@ export const AvailableTimeSlots = ({
           slotsPerDay.map((slots) => (
             <div key={slots.date} className="scroll-bar h-full w-full overflow-y-auto overflow-x-hidden">
               <AvailableTimes
+                className={`scroll-bar w-full overflow-y-auto overflow-x-hidden ${customClassnames?.availableTimeSlotsContainer}`}
```

Comment:
why do we have these new classess added here ? I mean this: scroll-bar w-full overflow-y-auto overflow-x-hidden

Reply:
My bad we previously didn't had the outside div and everything was being passed in the `AvailableTimes` className prop, might have inserted that at the time of resolving merge conflicts